dali/test/python/test_operator_readers_numpy.py

```diff
@@ -572,3 +573,41 @@ def test_numpy_reader_roi_error():
                     device, fortran_order, file_filter, roi_start, rel_roi_start, roi_end, \
                     rel_roi_end, roi_shape, rel_roi_shape, roi_axes, out_of_bounds_policy, \
                     fill_value
+
+def check_pad_last_sample(device):
+    with tempfile.TemporaryDirectory(prefix=gds_data_root) as test_data_root:
+        # create files
+        num_samples = 2
+        batch_size = 5
+        filenames = []
+        arr_np_list = []
+        last_file_name = None
+        for index in range(0, num_samples):
+            filename = os.path.join(test_data_root, "test_{:02d}.npy".format(index))
+            last_file_name = filename
+            filenames.append(filename)
+            create_numpy_file(filename, (5, 2, 8), np.float32, False)
+            arr_np_list.append(np.load(filename))
+        for _ in range(num_samples, batch_size):
+            arr_np_list.append(np.load(last_file_name))
+        pipe = NumpyReaderPipeline(path=test_data_root,
+                                files=filenames,
+                                file_list=None,
+                                file_filter=None,
+                                device=device,
+                                batch_size=batch_size,
+                                num_threads=4,
+                                device_id=0,
+                                pad_last_batch=True)
```

Comment:
PEP8 - align the indent to match the first argument.
```suggestion
        pipe = NumpyReaderPipeline(path=test_data_root,
                                   files=filenames,
                                   file_list=None,
                                   file_filter=None,
                                   device=device,
                                   batch_size=batch_size,
                                   num_threads=4,
                                   device_id=0,
                                   pad_last_batch=True)
```

Reply:
Done